Code/Engine/Foundation/System/Implementation/ProcessGroup.cpp

```diff
@@ -4,12 +4,8 @@
 // Include inline file
 #  if EZ_ENABLED(EZ_PLATFORM_WINDOWS)
 #    include <Foundation/System/Implementation/Win/ProcessGroup_win.h>
-#  elif EZ_ENABLED(EZ_PLATFORM_OSX)
-#    include <Foundation/System/Implementation/OSX/ProcessGroup_OSX.h>
-#  elif EZ_ENABLED(EZ_PLATFORM_LINUX) || EZ_ENABLED(EZ_PLATFORM_ANDROID)
-#    include <Foundation/System/Implementation/Posix/ProcessGroup_posix.h>
 #  else
-#    error "ProcessGroup functions are not implemented on current platform"
+#    include <Foundation/System/Implementation/ez/ProcessGroup_ez.h>
```

Comment:
I find the name ProcessGroup_ez.h a bit irritating, but finding a better one is indeed difficult. Maybe just ProcessGroup_other.h ?